src/internal/token_compressor_test.ts

```
@@ -0,0 +1,182 @@
+
+async function assertToken(
+  t: Deno.TestContext,
+  actual: Token,
+  expected: ExpectedToken,
+) {
+  if (expected.type !== undefined) {
+    await t.step(`should have type ${TokenType[expected.type]}`, () => {
+      assertStrictEquals(
+        actual.type,
+        expected.type,
+        `expected ${TokenType[expected.type!]}, got ${TokenType[actual.type]}`,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)